docs/layout-of-source-files.rst

```diff
@@ -28,6 +28,9 @@ it does include the supplied string in the :ref:`bytecode metadata <metadata>`.
 If you do not want to specify a license or if the source code is
 not open-source, please use the special value ``UNLICENSED``.
 
+.. note::
+    There might be a confusion between ``UNLICENSED`` (no usage allowed, not present in SPDX license list) and ``UNLICENSE`` (grants all rights to everyone). Check out `this discussion<https://github.com/ethereum/solidity/issues/12251>`_.
```

Comment:
What listen e is that for ? You see I’m been hacked and I have no clue what they have done and changed and deleted. Do you want to give me a hand and I’ll make it worth your while ?